tools/depcheck/pkg/cmd/trace/filter_test.go

```diff
@@ -0,0 +1,328 @@
+package trace
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/gonum/graph/concrete"
+
+	depgraph "github.com/openshift/origin/tools/depcheck/pkg/graph"
+)
+
+type testNode struct {
+	name     string
+	children []string
+}
+
+func getVendorNodes() []*testNode {
+	return []*testNode{
+		{
+			name: "github.com/test/repo/vendor/github.com/testvendor/vendor_one",
+			children: []string{
+				"github.com/test/repo/vendor/github.com/testvendor/vendor_one/misc",
+			},
+		},
+		{
+			name: "github.com/test/repo/vendor/github.com/testvendor/vendor_one/misc",
+			children: []string{
+				"github.com/test/repo/vendor/github.com/testvendor/vendor_three/vendor_three_misc",
```

Comment:
couldn't these be like: `prefix3/three`?  Just saying, it's a lot to read to get to that final bit of entropy.